applications/app/pages/ContentHtmlPage.scala

```diff
@@ -11,10 +11,10 @@ import views.html.fragments._
 import views.html.fragments.commercial.pageSkin
 import views.html.fragments.page.body.{bodyTag, breakingNewsDiv, mainContent, skipToMainContent}
 import views.html.fragments.page.head.stylesheets.{criticalStyleInline, criticalStyleLink, styles}
-import views.html.fragments.page.head.{fixIEReferenceErrors, headTag, titleTag, weAreHiring}
+import views.html.fragments.page.head._
```

Comment:
This is totally personal preference I think, but IMO keeping the imports explicit helps with future maintenance / finding usage etc.